--- old
+++ new
@@ -7,1 +7,1 @@
-Hi, I'm Polina, and this is my website. 
+Hi, I'm Polina, and this is a plot of digital soil I staked out for myself. Here are some quick facts about me:
@@ -9,1 +9,3 @@
-This website is named *issledovatel*, pronounced [ɪs⁽ʲ⁾ːˈlʲedəvətʲɪlʲ], which means *explorer* in Russian. I originally got this domain because I wanted to self-host a SearXNG search-engine instance, and thought the play on Internet Explorer was really clever. Needless to say, I did not end up using the domain for SearXNG, but instead repurposed it for my personal website. 
+- Job: software engineer
+- Education: UBC, BASc in Engineering Physics
+- 
@@ -11,0 +13,1 @@
+The website is called *issledovatel*, pronounced [ɪs⁽ʲ⁾ːˈlʲedəvətʲɪlʲ], which means *explorer* in Russian. I got this domain because I wanted to self-host a SearXNG search engine instance, and thought the play on Internet Explorer was really clever. Needless to say, I did not end up using the domain for SearXNG, but instead repurposed it for my personal site. 
@@ -12,0 +15,1 @@
+A bit about me: I'm a software engineer 
@@ -15,0 +19,1 @@
+