--- old
+++ new
@@ -7,1 +7,1 @@
-I'm Polina, and this is my plot of digital land. 
+Hi, I'm Polina, and this is my website. 
@@ -9,1 +9,6 @@
-This website is called issledovatel, pronounced [ɪs⁽ʲ⁾ːˈlʲedəvətʲɪlʲ]. 
+This website is named *issledovatel*, pronounced [ɪs⁽ʲ⁾ːˈlʲedəvətʲɪlʲ], which means *explorer* in Russian. I originally got this domain because I wanted to self-host a SearXNG search-engine instance, and thought the play on Internet Explorer was really clever. Needless to say, I did not end up using the domain for SearXNG, but instead repurposed it for my personal website. 
+
+
+
+
+