--- old
+++ new
@@ -11,1 +11,1 @@
-This website is called *issledovatel*, pronounced [ɪs⁽ʲ⁾ːˈlʲedəvətʲɪlʲ], which means *explorer* in Russian. I got this domain because I wanted to self-host a SearXNG search engine instance, and thought the play on *Internet Explorer* was very clever. Needless to say, I did not end up using the domain for its intended purpose, but instead attached it to my personal site. 
+This website is called *issledovatel*, pronounced [ɪs⁽ʲ⁾ːˈlʲedəvətʲɪlʲ], which means *explorer* in Russian. I got this domain because I wanted to self-host a SearXNG search engine instance, and thought the play on *Internet Explorer* was very clever. Needless to say, I did not end up using the domain for its intended purpose, but instead  to my personal site. 